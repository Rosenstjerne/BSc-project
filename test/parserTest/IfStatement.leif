var int a,b;
a = 1;
b = 2;

if (a == 1)
    print(3);