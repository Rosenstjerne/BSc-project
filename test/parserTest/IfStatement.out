function null main() {
	var int a, b;
	
	a = 1;
	b = 2;
	if((a == 1))
	#begin if
		print(3);
	#end if
	
}
